packages/odyssey-lint-staged/src/index.js

```diff
@@ -11,7 +11,7 @@
  */
 
 module.exports = {
-  "*": "yarn prettier --check . --ignore-path .gitignore",
-  ".scss": "stylelint",
-  ".js,.jsx,.ts,.tsx": "eslint .",
+  "*": "prettier --ignore-unknown --ignore-path .gitignore --write",
+  "*.scss": "stylelint",
+  "*.{js,jsx,ts,tsx}": "eslint",
```

Comment:
I experimented with adding type checking here but think we'll need to bit a bit more nuanced with our glob patterns to target specific package `tsconfig.json` files